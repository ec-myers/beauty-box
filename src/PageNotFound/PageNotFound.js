import React from 'react';
import { Link } from 'react-router-dom';
import './PageNotFound.scss';

const PageNotFound = () => {
  return (
    <div className='PageNotFound'>
      <img src={require('../Images/page-not-found.png')} alt='404 error, page not found'/>
      <h3>Looks like you've gotten lost...</h3>
      <h4>The page you're looking for doesn't exist or has been moved.</h4>
      <Link to='/'><button>GO HOME</button></Link>
    </div>
  )
}

export default PageNotFound;
