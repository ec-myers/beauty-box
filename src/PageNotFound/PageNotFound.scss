$background-color: rgb(255, 217, 218);
$font-color: rgb(5,29,65);
$button-hover-color: darken(rgb(191, 192, 192), 8%);

.PageNotFound {
  width: 100%;
  text-align: center;
  margin-top: 40px;

  h3 {
    font-size: 2em;
    margin: 10px;
  }

  h4 {
    font-size: 1.5em;
    margin: 10px;
  }

  button {
    background-color: $font-color;
    border: none;
    color: white;
    cursor: pointer;
    font-size: 1em;
    font-weight: bold;
    height: 45px;
    margin: 30px;
    width: 185px;

    &:hover {
      background-color: $button-hover-color;
    }
  }
}