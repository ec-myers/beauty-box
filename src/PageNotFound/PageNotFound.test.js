import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from './PageNotFound';

describe('PageNotFound', () => {
  it('should match snapshot', () => {
    let wrapper = shallow(<PageNotFound />);

    expect(wrapper).toMatchSnapshot();
  });
});