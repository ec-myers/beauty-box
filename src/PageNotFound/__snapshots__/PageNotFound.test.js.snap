// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound should match snapshot 1`] = `
<div
  className="PageNotFound"
>
  <img
    alt="404 error, page not found"
    src="page-not-found.png"
  />
  <h3>
    Looks like you've gotten lost...
  </h3>
  <h4>
    The page you're looking for doesn't exist or has been moved.
  </h4>
  <Link
    to="/"
  >
    <button>
      GO HOME
    </button>
  </Link>
</div>
`;
